config/software/protobuf-py.rb

```diff
@@ -45,6 +45,6 @@
         delete "#{install_dir}/embedded/lib/libprotoc.*"
         delete "#{install_dir}/embedded/bin/protoc"
     else
-        command "#{install_dir}/embedded/bin/pip install protobuf==#{rhel_pip_version}"
+        pip "install protobuf==#{rhel_pip_version}"
```

Comment:
there's a `pip install` command 4 lines above that you may want to replace too :)